golang/cosmos/x/vstorage/vstorage_test.go

```diff
@@ -274,13 +274,13 @@ func TestEntries(t *testing.T) {
 		want string
 	}
 	cases := []testCase{
-		{path: "key1", want: `[["child1",null]]`},
+		{path: "key1", want: `[["child1"]]`},
```

Comment:
It was requested in https://github.com/Agoric/agoric-sdk/pull/7489#discussion_r1175603227 , and I went with `null` at the time but reconsidered when I got to this PR because it was unnecessary overhead.